microfrontends/app-builder-menu/src/utils/getAPIEndpoint.ts

```diff
@@ -0,0 +1,11 @@
+import { MfeConfig } from '../types/globals';
+
+export type APIEndpointFields = "navigation"
+
+export function getAPIEndpoint(fieldName: APIEndpointFields, config: MfeConfig) {
+  const { url } = config.systemParams.api[fieldName];
```

Comment:
yes, I agree. I've not added this control because `config.systemParams.api` is always present because is passed by the `shell` even if there isn't an API.